mkdocs.yml

```diff
@@ -8,21 +8,21 @@ site_url: https://physiopy.github.io
 copyright: Copyright &copy; 2020
 
 nav:
-    - Home: docs/index.md
+    - Home: index.md
     - Libraries:
-      - phys2bids: docs/libraries/phys2bids.md
-      - phys2denoise: docs/libraries/phys2denoise.md
-      - peakdet: docs/libraries/peakdet.md
+      - phys2bids: 'https://phys2bids.readthedocs.io/en/latest/'
```

Comment:
Wait, I thought we'd have an introductory page with the link to the "real" `phys2bids` documentation in it.

Reply:
Yeah, well I gave it a try. Sorry, i didn't mention that in my comments. I was thinking of suggesting we describe libraries in the Home `index.md`

Reply:
we can still change it before merging

Reply:
> Yeah, well I gave it a try. Sorry, i didn't mention that in my comments. I was thinking of suggesting we describe libraries in the Home `index.md`

That's a good option too. I like the idea. Then the tabs will direct to the "real" docs of each package. I like it!